crates/bevy_ecs/src/event.rs

```diff
@@ -34,6 +36,7 @@ pub trait Event: Send + Sync + 'static {}
 /// sent to the point it was processed. `EventId`s increase montonically by send order.
 ///
 /// [`World`]: crate::world::World
+#[cfg_attr(feature = "bevy_reflect", derive(Reflect))]
 pub struct EventId<E: Event> {
```

Comment:
Should this be a ReflectValue instead? :thinking:

Reply:
Hmm. It's definitely value-like. I can make that change if others think it's useful.

```suggestion
#[cfg_attr(feature = "bevy_reflect", reflect_value]
pub struct EventId<E: Event> {
```

Reply:
I think it depends. Do we ever want to access the `id`? If so, then we should probably leave it as is.

Reply:
Yeah, you'll want to access the ID. I'll ignore the PhantomData.